package.json

```diff
@@ -23,38 +23,59 @@
     "postinstall": "typings install",
     "start": "gulp docs && webpack-dev-server --devtool eval --inline --hot --progress --colors",
     "build": "gulp",
+    "pretest": "rm -rf coverage",
+    "test": "karma start",
+    "posttest": "npm run coverage",
+    "coverage": "npm run coverage:remap && npm run coverage:report",
+    "coverage:remap": "remap-istanbul -i coverage/coverage.json -o coverage/coverage.json -t json -e node_modules,tests,karma.entry.ts",
+    "coverage:report": "istanbul report",
     "prepublish": "npm run build"
   },
   "dependencies": {
     "react-tether": "0.5.2"
   },
   "peerDependencies": {
     "tether": "^1.3.7",
-    "react": "^15.3.1",
-    "react-dom": "^15.3.1"
+    "react": "^15.3.2",
```

Comment:
https://nodejs.org/en/blog/npm/peer-dependencies/

So we don't want to force users of our library to use a fixed version of react, we only want to warn them on the fact they need to install it and they need to have at least the version specified there.